internal/templates/01-update.yaml.tmpl

```diff
@@ -0,0 +1,11 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+{{- range $resource := .Resources }}
+{{- if eq $resource.KindGroup "secret." -}}
+  {{continue}}
+{{- end -}}
+{{- if not $resource.Namespace }}
+- script: '${KUBECTL} patch {{ $resource.KindGroup }}/{{ $resource.Name }} --type=json -p "[{"op": "add", "path": "/spec/forProvider/{{ $resource.UpdateField }}", "value": {{ $resource.UpdateValue }}}]"'
```

Comment:
We can define a provider-level default for the parameter to be used in resource update tests. We could define this default in the reusable job definition as follows:
```yaml
jobs:
  open-bump-pr:
    uses: ...
    with:
        update-test-parameter: ...
```

Reply:
Done!